pkg/nodes/node.go

```diff
@@ -77,11 +106,79 @@ func (builder *Builder) Exists() bool {
 
 	glog.V(100).Infof("Checking if node %s exists", builder.Definition.Name)
 
+	var err error
+	_, err = builder.Get()
+
+	return err == nil || !k8serrors.IsNotFound(err)
+}
+
+// Get fetches existing node from the cluster.
+func (builder *Builder) Get() (*Builder, error) {
```

Comment:
You do not need a get method for resources that provide their own get method (builder.Object, err = builder.apiClient.CoreV1Interface.Nodes().Get)